package.json

```diff
@@ -30,8 +30,19 @@
   ],
   "author": "Daniel Cousens",
   "license": "MIT",
+  "files": [
+    "index.js",
+    "README.md"
+  ],
   "devDependencies": {
+    "benchmark": "^2.0.0",
     "mocha": "*",
     "standard": "*"
+  },
+  "standard": {
+    "global": [
```

Comment:
Personally I don't like this,  it means `describe` and `it` are globals in `index.js` too, not just the tests.